git/remotes_in_git.md

```diff
@@ -0,0 +1,464 @@
+
+Now that we have two repositories, we need a diagram like this:
+
+![git-freshly-made-github-repo-05](./assets/media/remotes_step_01_images/git-freshly-made-github-repo-05.svg)
+
+<div class = "important">
+Note that our local repository still contains our earlier work on mars.txt, but the remote repository on GitHub appears empty as it doesn’t contain any files yet.
+</div>
+
+### Quiz: GitHub license and README files
+
+In this module, we learned about creating a remote repository on GitHub, but when you initialized your GitHub repo, you didn’t add a README.md or a license file. If you had, what do you think would have happened when you tried to link your local and remote repositories?
+
+[[GitHub license and README files]]
+***
+<div class = "answer">
+In this case, we’d see a merge conflict due to unrelated histories. When GitHub creates a README.md file, it performs a commit in the remote repository. When you try to pull the remote repository to your local repository, Git detects that they have histories that do not share a common origin and refuses to merge.
+
+```console
+$ git pull origin main
+```
+
+```output
+warning: no common commits
+remote: Enumerating objects: 3, done.
+remote: Counting objects: 100% (3/3), done.
+remote: Total 3 (delta 0), reused 0 (delta 0), pack-reused 0
+Unpacking objects: 100% (3/3), done.
+From https://github.com/vlad/planets
+ * branch            main     -> FETCH_HEAD
+ * [new branch]      main     -> origin/main
+fatal: refusing to merge unrelated histories
+```
+</div>
+***
+
+## Connect local to remote repository
+
+Now we connect the two repositories. We do this by making the GitHub repository a remote for the local repository. The home page of the repository on GitHub includes the URL string we need to identify it:
+
+![github-find-repo-string-01](./assets/media/remotes_step_02_images/github-find-repo-string-01.png)
+
+Click on the ‘SSH’ link to change the protocol from HTTPS to SSH.
+
+![github-change-repo-string-02](./assets/media/remotes_step_02_images/github-change-repo-string-02.png)
+
+Copy that URL from the browser, go into the local planets repository, and run this command:
+
+
+```console
+$ git remote add origin git@github.com:vlad/planets.git
+```
+
+Make sure to use the URL for your repository rather than Vlad’s: the only difference should be your username instead of `vlad`.
+
+`origin` is a local name used to refer to the remote repository. It could be called anything, but `origin` is a convention that is often used by default in git and GitHub, so it’s helpful to stick with this unless there’s a reason not to.
+
+We can check that the command has worked by running `git remote -v`:
+
+```console
+$ git remote -v
+```
+
+```output
+origin   git@github.com:vlad/planets.git (fetch)
+origin   git@github.com:vlad/planets.git (push)
+```
+
+We’ll discuss remotes in more detail in the next episode, while talking about how they might be used for collaboration.
+
+### SSH
```

Comment:
Should SSH become its own module that can then be a prereq here? It just seems like a pretty lengthy detour from the main content of the module.